examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -44,7 +44,7 @@ namespace picongpu
 
         /* TWTS E-fields need to be initialized on host,
          *  so they can look up global grid dimensions. */
-        PMACC_ALIGN(twtsFieldE, const templates::twts::EField);
+        const templates::twts::EField twtsFieldE;
```

Comment:
Because using `PMACC_ALIGN` roughly doubles the number of registers used in the actual compiled kernel. It was this keyword that prevented some of my simulations from starting. Since the `twtsFieldE`-Object is already internally aligned, further memory alignment is also not really necessary. I will include an according comment, so that this gimmick survives the next round of refactoring.